app/rdf/query-dimension-values.ts

```diff
@@ -165,7 +165,7 @@ CONSTRUCT {
     }
   } UNION`
   } {
-    {
+    { #pragma evaluate on
```

Comment:
What does that do ?

Reply:
This suggestion comes from @Rdataflow and seems to optimize the execution time – exact description is "suggests to evaluate the scope at query optimization time", see https://docs.stardog.com/query-stardog/query-hints

Reply:
@ptbrowne see https://docs.stardog.com/query-stardog/query-hints#general-query-optimization-hints

<html>
<body>
<!--StartFragment-->
evaluate | on,off | scope | Suggests to evaluate the scope at query optimization time.
-- | -- | -- | --


<!--EndFragment-->
</body>
</html>